static/app/components/deprecatedSmartSearchBar/index.tsx

```diff
@@ -658,7 +658,8 @@ class DeprecatedSmartSearchBar extends Component<DefaultProps & Props, State> {
 
       let offset = filterTokens[0]!.location.end.offset;
       if (token) {
-        const tokenIndex = filterTokens.findIndex(tok => tok === token);
+        // @ts-expect-error: Mismatched types
+        const tokenIndex = filterTokens.indexOf(token);
```

Comment:
There's a type error here i didn't dig into, the type of `token` is huge.

That the types don't match might indicate an (existing?) problem